lib/client/js/ui/controls/Notifications.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * The UI Control that manages the Notifications section of
+ * the BeansContent
+ *
+ * @class
+ */
+var Notifications = (function () {
+    /**
+     * @alias Notifications
+     */
+    function Notifications() {
+        // turn the arguments of this constructor into an array
+        var args = Array.prototype.slice.call(arguments);
+
+        /*
+         * @see {@link http://tobyho.com/2010/11/22/javascript-constructors-and/}
+         */
+
+        // Apply the Control constructor to the Notifications instance with the input arguments
+        Control.apply(this, args);
+    }
```

Comment:
Question: So, I've been noticing that constructors don't have a semicolon after their closing bracket. Yet, every other function seems to have one. Is this JS standard?